components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -144,13 +130,11 @@ impl TransitionProperty {
         //
         // FIXME: This should handle aliases too.
         pub enum StaticId {
-            All,
             Longhand(LonghandId),
             Shorthand(ShorthandId),
         }
         ascii_case_insensitive_phf_map! {
             static_id -> StaticId = {
-                "all" => StaticId::All,
                 % for prop in data.shorthands_except_all():
```

Comment:
this shouldn't be `except_all` anymore.

Reply:
Good catch.

New run with that fixed: https://treeherder.mozilla.org/#/jobs?repo=try&revision=dd4029371d115739dcdfe25b22401b32b531d242